tests/unit/k8s/test_scrape_configuration.py

```diff
@@ -75,6 +75,8 @@
BVMB8GA1UdDgQYBBYEFFz/sE6F0zH4E6ahq5BEx2bY\no0GdMCEGA1UdIwQaMBiAFgQUXP+wToXTMfgTpqGrkETHZtijQZ0wDwYDVR0TAQH/\nBAUwAwEB/zANBgkqhkiG9w0BAQsFAAOCAQEAUNqmTqlJhUEW+1S1cAMwtkLGZ9S9\nHIZ/u6TeJKx9NvDYBW/dJzMkfNC5lKT18Hyat7Qrnmpa+cXKra+VPt/18eRylZgI\nR5UpTaaZF48jkisYcxm7hGoR7G3vNd1cUKweRuDR0bRrJOZ9RQacsaWSFMX2Afv/\nQl0XsvCUHXL1R3f2eEquOFlOqd4YnlPA5b0qo20ezDYbOjKS648G8t6r0JvNXU1G\nHRcCfSN3uMuP7dEYylzP/aECFMcP/wdl9GyuOJhyNCmucQRPiNM5RQp71CmDXpUN\nFP+svfCC/swN8ZSxqYxubVL/kyUAsPk81IO7zOrbdd9gJEuRKWoWa/eWxw==\n-----END CERTIFICATE-----", "chain": ["-----BEGIN CERTIFICATE-----\nMIIDVzCCAj+gAwIBAgIURIhdtuPZiz2sebg5YgDSDoJRcnwwDQYJKoZIhvcNAQEL\nBQAwOTELMAkGA1UEBhMCVVMxKjAoBgNVBAMMIXNlbGYtc2lnbmVkLWNlcnRpZmlj\nYXRlcy1vcGVyYXRvcjAeFw0yMzA3MTMyMzI1MDhaFw0yNDA3MTIyMzI1MDhaMDkx\nCzAJBgNVBAYTAlVTMSowKAYDVQQDDCFzZWxmLXNpZ25lZC1jZXJ0aWZpY2F0ZXMt\nb3BlcmF0b3IwggEiMA0GCSqGSIb3DQEBAQUAA4IBDwAwggEKAoIBAQCllekttoc9\nFuO4byyyemRw+Jb0ptfeFNX4qXPhl/H3TIJkkOk9LX0AJr/0sIICfqYp9IBvcyaR\n7ECHuZsQUGKq3UR4DCmDwprjfBAA/7tk0cbuORPeBOoAJLz8/ubllbJnaCCdW+bb\nhB5w+0IZ2dtIPNWc3TAwyjv6JkwDrR197+ievZ8rFFE/0v+40baMA+tMdAgB4EuE\nvnDIZ/0Js0Nv6Lkc8Ga+3BJieH9eSC6LDDXlWv5u59NrjCbr7HLZxyqSISavFFnQ\n08k/luuQRfx4an0qjOOPNk6k2eDsRTR2Ov/aiwkw4mI+IOKP5SVEgVKJx80XXXCu\nAcliBMpJ2PnhAgMBAAGjVzBVMB8GA1UdDgQYBBYEFFz/sE6F0zH4E6ahq5BEx2bY\no0GdMCEGA1UdIwQaMBiAFgQUXP+wToXTMfgTpqGrkETHZtijQZ0wDwYDVR0TAQH/\nBAUwAwEB/zANBgkqhkiG9w0BAQsFAAOCAQEAUNqmTqlJhUEW+1S1cAMwtkLGZ9S9\nHIZ/u6TeJKx9NvDYBW/dJzMkfNC5lKT18Hyat7Qrnmpa+cXKra+VPt/18eRylZgI\nR5UpTaaZF48jkisYcxm7hGoR7G3vNd1cUKweRuDR0bRrJOZ9RQacsaWSFMX2Afv/\nQl0XsvCUHXL1R3f2eEquOFlOqd4YnlPA5b0qo20ezDYbOjKS648G8t6r0JvNXU1G\nHRcCfSN3uMuP7dEYylzP/aECFMcP/wdl9GyuOJhyNCmucQRPiNM5RQp71CmDXpUN\nFP+svfCC/swN8ZSxqYxubVL/kyUAsPk81IO7zOrbdd9gJEuRKWoWa/eWxw==\n-----END CERTIFICATE-----", "-----BEGIN CERTIFICATE-----\nMIIDRDCCAiygAwIBAgIUPf3TRo7siuI9UP63UKsRF/4sUakwDQYJKoZIhvcNAQEL\nBQAwOTELMAkGA1UEBhMCVVMxKjAoBgNVBAMMIXNlbGYtc2lnbmVkLWNlcnRpZmlj\nYXRlcy1vcGVyYXRvcjAeFw0yMzA3MTMyMzM2MzdaFw0yNDA3MTIyMzM2MzdaMEEx\nEDAOBgNVBAMMB2FnZW50LzAxLTArBgNVBC0MJGJhMjk3MDhkLTEwMTYtNGQ2OS04\nOTRjLTgwOGYxYjZkMzI5NzCCASIwDQYJKoZIhvcNAQEBBQADggEPADCCAQoCggEB\nAJTck0+65BgohrJYrfQJKhkqiSZ1RD3upOHd69xzXda+5SJ+tjVU8aKTrY7BMyzD\n97mmeCZMryVrwa3JuoQR2ktf4Wq7DqRjG+YKFrN+MjlXu5ljTNvDZUheteahVUr9\nPDB67zgrhgc8TgorcESFlLSVQOQrMx3BEPN+PSFDqUme+0zQ5G9qCAY785ikAn7J\nGwd2tONYvrG1jA1rRw8SsF6CVZsgFgbHpfaj4Oq+yygwHXA1i0s7rf9DlUxrYaZC\nth2TzEPV9Iu0HNrJzWHJxDpeOvTptSZs5mBKSgCVW9nIaIuPWj8sEwKW9YVVlNEx\ngyjHLST4o340lKVqk2qXXE0CAwEAAaM8MDowOAYDVR0RBDEwL4ItYWdlbnQtMC5h\nZ2VudC1lbmRwb2ludHMubG1hLnN2Yy5jbHVzdGVyLmxvY2FsMA0GCSqGSIb3DQEB\nCwUAA4IBAQA8sD4zDFp0+cmv6acbSWiAVn/qZBcz0qpqmOqeUwd1IvcsczOwSzq5\n8Cf1TCJuqJFiZgyiAVNRXF8MxNXrYYB/LI5L52AeAwsv6Fpo+x8tqkzNwbpY+2aA\n1Q2Toc5/mCCSH9rzsznCjHnR1yUcEt/CzI9wfwdH0F5v5u7onrx0XXtnEiDTQIDV\nPQxGlI2xPdisqdLmFZgCjOpANRfAiBnQY1pLkH3rwHWHDsvb1jTy8UEpzYINu0vd\nogI1GMccHAq4MM7tRjmSuOL9XqgTEEvSJeJDSexDe0iWC3AYCat79tUV9gGfpkK/\nA+GdAqYgyk2tL0iL4WTIkg0t52pgkXPf\n-----END CERTIFICATE-----"]}]"""
```

Comment:
can we find a way to have "foobar" in here, instead of this humongous realistic-looking cert? Or do we fail some validation we can't easily mock?

Reply:
I just pasted from relation data. I'll check.

Reply:
Fixed.